HousingFinanceInterimApi.Tests/V1/UseCase/MoveHousingBenefitFileUseCaseTests.cs

```diff
@@ -183,28 +183,33 @@ public async Task UCCallsGoogleClientServiceCopyFileInDriveMethodForEachValidAca
             var academyNewFilesCount = 2;
             var academyAlreadyCopiedFilesCount = 3;
 
+            // Create 2 new files
             var academyFolders = RandomGen.CreateMany<GoogleFileSettingDomain>(quantity: 1);
             var academyNewFiles = RandomGen.GoogleDriveFiles(filesValidity: true, count: academyNewFilesCount);
 
-            var oldNameFile1 = "07042022_Something_Academy_21042022";
-            var oldNameFile2 = "20012023_Something_Academy_03022023";
-            var oldNameFile3 = "15082022_Something_Academy_29082022";
+            var newFiles = academyNewFiles as File[] ?? academyNewFiles.ToArray();
+            newFiles[0].Name = "07042022_Something_Academy_21042025";
+            newFiles[1].Name = "07042022_Something_Academy_03022026";
+            newFiles[0].CreatedTime = new DateTime(2023, 4, 1);
+            newFiles[1].CreatedTime = new DateTime(2023, 4, 11);
```

Comment:
The naming convention of the Academy file takes the following structure:
`rentpost_[startDate]_to[endDate].dat`

Please use this structure in the naming of the file on lines 191 and 192.

Also, just out of curiosity, can we confirm that the [startDate] is not used anywhere. Just to be sure.